common/types/channel.d.ts

```diff
@@ -10,5 +10,5 @@ export interface ChannelOptions {
     algorithm: string;
     encrypt: Function;
     decrypt: Function;
-  };
+  } | null;
```

Comment:
What does this do?

Reply:
Just types it so that `channelCipher` can be null. There's a place in `channel.ts` which sets options.channelCipher to null so this had to be nullable.